notifique/src/main/res/drawable/thumb.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <padding
```

Comment:
what does this padding do?

Reply:
I think it helps with selecting the button, idk

Reply:
removed.